arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -0,0 +1,1029 @@
+          "enum": [
+            "Default",
+            "WaitingForNodes",
+            "Deploying",
+            "BaselineUpgrade",
+            "UpdatingUserConfiguration",
+            "UpdatingUserCertificate",
+            "UpdatingInfrastructure",
+            "EnforcingClusterVersion",
+            "UpgradeServiceUnreachable",
+            "Deleting",
+            "ScaleUp",
+            "ScaleDown",
+            "AutoScale",
+            "Ready",
+            "Failed"
+          ]
+        },
+        "clusterEndpoint": {
+          "description": "The endpoint for the cluster connecting to servicefabric resource provider",
+          "readOnly": true,
+          "type": "string"
+        },
+        "clusterCodeVersion": {
+          "description": "The ServiceFabric code version, you don't need to set it, when you choose upgradeMode as Automatic",
+          "type": "string"
+        },
+        "certificate": {
+          "description": "The cluster certificate",
+          "$ref": "#/definitions/CertificateDescription"
+        },
+        "reliabilityLevel": {
+          "description": "The reliability level for the cluster",
+          "type": "string",
+          "enum": [
+            "Bronze",
+            "Silver",
+            "Gold",
+            "Platinum"
+          ]
+        },
+        "upgradeMode": {
+          "description": "The cluster upgrade mode",
+          "type": "string",
+          "enum": [
+            "Automatic",
+            "Manual"
+          ]
+        },
+        "clientCertificateThumbprints": {
+          "description": "The client certificate thumbprints",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientCertificateThumbprint"
+          }
+        },
+        "clientCertificateCommonNames": {
+          "description": "The certificate common name and issurer thumprint for the client certificate",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientCertificateCommonName"
+          }
+        },
+        "fabricSettings": {
+          "description": "The faric setting collection",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SettingsSectionDescription"
+          }
+        },
+        "reverseProxyCertificate": {
+          "description": "The certificate for the reverse proxy",
+          "$ref": "#/definitions/CertificateDescription"
+        },
+        "managementEndpoint": {
+          "description": "The cluster management endpoint address",
+          "type": "string"
+        },
+        "nodeTypes": {
+          "description": "The arary of the nodeType",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NodeTypeDescription"
+          }
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the cluster",
+          "type": "string",
+          "enum": [
+            "Default",
```

Comment:
Creating is missing, what is meant by Default here, do you send default state out.